TESTING.md

```diff
@@ -168,6 +168,23 @@ automatically.
 * All commands are visible in the Commands panel and can be selected.
 * All commands works after selection.
 
+### General commands availability when selection is active
```

Comment:
<img width="376" alt="image" src="https://github.com/sourcegraph/jetbrains/assets/19799111/93a93363-5bd9-4418-bf5a-518600799406">

It looks like this for me. Is it expected? 

Probably a space distance b/w the blue box and the last character in the line would make it better. Can you link Daniel's design?

Reply:
cc: @danielmarquespt 

I actually already added 2 extra spaces of distance at the end of the string. Daniel, can you try the branch again and see if the UI is doing what you want it to do?